things/postgres/init.go

```diff
@@ -57,14 +57,14 @@ func migrateDB(db *sqlx.DB) error {
 						owner    VARCHAR(254),
 						key      VARCHAR(4096) UNIQUE NOT NULL,
 						name     VARCHAR(1024),
-						metadata JSON,
+						metadata JSONB,
```

Comment:
Please consider adding a separate migration for type change, rather than changing data type this way.

Reply:
@dusanb94 I do not see the need to support `JSON` anymore. Pre `1.0` can stil break APIs and we are not in obligation to drag with us unneeded field.

Reply:
It would be easier for us because we wouldn't need to change schema manually (no need to drop/change in Docker volumes, nor local DB). I agree we can remove that migration before production because this is <1.0, but adding even temporary migration will help (especially since it's done only once).